index.html

```diff
@@ -207,15 +262,17 @@ <h2  class="contactme" >Contact me </h2>
         </div>
             <div class="form-group">
                  <input class="texform" type="text"   id="Name"    classe="form-control" placeholder="Enter Your Name" minlength="4" maxlength="30" required />
+            </div>
+            <div class="form-group">          
                 <input class="texform"  type="email"  id="email"  name="email"   placeholder="Enter Your Email"  required  />
-            </div >
+            </div>
             <div class="form-group2">
-                <textarea class="comment"  name="comment-section" id="comment"  maxlength="500"s  placeholder="Write your message here..."  required ></textarea>
+                <input class="texform2"  name="comment"  id="comment"   cols="50"   rows="6"  placeholder="Write your message here..."  required  />
             </div>
-              
-            <div class="buttonsee2">
+            <div class="buttonsee2" id="contactbuttonid">
                 <button class="contactbutton"> Get It Touch </button>
```

Comment:
Kindly make sure if your submit button is aligned well like the provided design.
Your design:
<img width="1440" alt="Screenshot 2022-12-16 at 9 59 35 PM" src="https://user-images.githubusercontent.com/25887253/208173974-8d5dc84d-f62f-4537-bb11-2eed1ff4402b.png">
Figma template:
<img width="371" alt="Screenshot 2022-12-16 at 10 26 04 PM" src="https://user-images.githubusercontent.com/25887253/208173936-26c55bbd-d033-4f70-9106-3ed2a571159c.png">